readme.txt

```diff
@@ -45,6 +45,14 @@ Please visit our [wiki](https://github.com/alleyinteractive/apple-news/wiki) for
 
 == Changelog ==
 
+= 1.2.3 =
+* Allowed mapping themes to Apple News sections
+* Added support for videos in feed
+* Added support for maturity rating
+* Added support for cover art
+* Added support for the Facebook component
+* Added support for captions in galleries
```

Comment:
Bugfix for invalid JSON errors caused by non-breaking spaces and other Unicode separators.